jetstream/metric.py

```diff
@@ -0,0 +1,53 @@
+import attr
+
+import mozanalysis.experiment
+import mozanalysis.metrics
+
+from typing import Optional
+
+
+@attr.s(auto_attribs=True, frozen=True, slots=True)
+class Metric:
```

Comment:
`class Metric(mozanalysis.metrics.Metric)` doesn't work. mypy complains that the fields defined in the mozanalysis `Metrics` class are missing and when passing `Metric` to a Dask worker the `analysis_basis` value gets erased.

Reply:
That's fine; I think we don't really want to deal with cross-package inheritance here anyway.